capture-traffic/capture-browser-traffic.md

```diff
@@ -44,6 +44,26 @@ As a result, Fiddler Everywhere starts an independent browser instance preconfig
 
 The independent browser capturing mode supports Chromium-based browsers like Google Chrome, Microsoft Edge, Brave, and Vivaldi. You can set your preferred browser through **Settings > Browsers > Manually choose path to browser**. By default, Fiddler will try to automatically detect the path to your locally installed Google Chrome browser.
 
+
+## Persisting Data
+
+The independent browser capturing mode allows you to create a custom directory where the browser data persists. That way, you can reuse the same custom browser instance and easily reuse data like entered URLs, cookies, form data, authentication details, etc.
```

Comment:
easily reuse data like entered URLs, cookies, form data, authentication details, etc. - to omit doubling the "reuse" word, you can change this to something like: easily keep your settings, like entered URLs, cookies, form data, authentication details, etc, between browser starts.